agoraStreaming/src/Screens/Profile/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {FC} from 'react';
+import {Image, Text, TouchableOpacity, View} from 'react-native';
+
+import {styles} from './style';
+import {ProfileScreenProps} from './types';
+
+export const ProfileScreen: FC<ProfileScreenProps> = () => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.wrapperProfile}>
+        <View style={styles.profile}>
+          <Image
+            style={styles.imageUser}
+            source={require('../../../assets/images/user.png')}
```

Comment:
try import userPng from '../../../assets/images/user.png' and use required(userPng)